tasks/main.yml

```
@@ -55,8 +60,15 @@
     - yagna_datadir
     - provider_datadir
     - plugins
+    - installer_resources
   notify: Restart provider
 
+- name: Extract installer resources
+  unarchive:
+    src: "{{ ya_provider_installer_resources_url }}"
+    dest: "{{ ya_provider_subnet_dir }}/installer_resources"
```

add to `vars/main.yml`
```yaml
ya_provider_resources_dir: "{{ ya_provider_releases_dir }}/installer_resources-{{ ya_provider_installer_resources_version }}/"
```

Create ya_provider_resources_dir directory before.

```suggestion
    dest: "{{ ya_provider_resources_dir }}/"
    creates: "{{ ya_provider_resources_dir }}/certs/GOLEM-CA.pem"
```

---

1. Maybe it should be just `creates: "{{ ya_provider_resources_dir }}/certs` ?
2. Will not block unarchive when we use new version `ya-installer-resources`?

---

1. It can be.
2. No, because `ya_provider_resources_dir` contains `ya_provider_installer_resources_version`. It will prevent downloading already downloaded resources of specified version.